frontend/javascripts/admin/user/user_list_view.tsx

```diff
@@ -474,7 +474,7 @@ class UserListView extends React.PureComponent<Props, State> {
               title="Email"
               dataIndex="email"
               key="email"
-              width={350}
+              width={320}
```

Comment:
The user list is slightly too wide on a 15" screen and causes scrollbars to appear. Slight making the default width of the email column smaller fits the table inside the available screen space